UnityClient/Assets/Scripts/PauseMenu.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PauseMenu : MonoBehaviour
+{
+
+    public static bool GamePaused = false; /*Declare GamePaused variable, initially set to false*/
```

Comment:
`static` is innately evil. Please rework this to not be static. Also public fields are bad practice. Please revise into a full or auto property.